### ISSUE TITLE
Support for Custom Post Types

### ISSUE THREAD

**Issue by SiobhyB** [wordpress-mobile/WordPress-Android#7216]
In a review of mobile feedback from the past six months (p77Llu-9N5-p2), support for custom post types was a common request.

Portfolios, in particular, are popular and it would be convenient for users who keep a portfolio-based site to be able to manage it directly from the app. 

The iOS version of this request is available at wordpress-mobile/WordPress-iOS#500.

**Comment by BigActual** [comment]
Client requested ability to edit CPT's via Android app today.

**Comment by sarahblackstock** [comment]
Portfolios requested in app support. 

internal ref: #262563-h

**Comment by SiobhyB** [comment]
Requested again in 285518-h.

**Comment by formosattic** [comment]
Portfolios requested in 1341215-zen.

**Comment by BigActual** [comment]
CPT editing requested by client again today.

**Comment by SiobhyB** [comment]
Three star review:

> I can not see the Portfolio section and that is a huge mistake since my blogs are only portfolio.

**Comment by SiobhyB** [comment]
Four star review:

> It's super. Plain, easy and super. One request only: custom post types guys. Nowadays, it's not just posts and pages. If you give content access to custom post types, would boost the functionality 100%! Keep the awesome work guys!

**Comment by donalirl** [comment]
Request for portfolio post type in #1570541-zd

**Comment by SiobhyB** [comment]
From a three star review:

> No Projects/Portfolio menu. Can you add it? It is essential.

**Comment by gemmagarner** [comment]
Requested [on Twitter](https://twitter.com/makemysushi/status/1114897681011367936).

**Comment by thehenrybyrd** [comment]
Another request from 1946176-zen
> Please add an option to ADD A PORTFOLIO PROJECT

**Comment by rezzap** [comment]
From a one star review:

> adding or editing of portfolio projects Impossible 😵

**Comment by designsimply** [comment]
> Does it incorporates custom post types yet?
> 
> <sup>2-star review by Pete Notsae 8/22 5:44 PM Huawei Nova 3e Android 9</sup>

<sup>play-store-eviewid=gp:AOqpTOEN6VUIMqMed6Rj9BWVQs5CXVfvRl-OHp2BpoXYy3uhKruwEOCuCyCKAGR8DUo1tN9nAB6R9c3xJnrC7A</sup>

**Comment by rezzap** [comment]
From a 3 star review:

> Think it is a good app. Works fine in itself. What I personally would like to see added is the ability to also create custom post types in the app. For my website I use custom post types, but they cannot be created from the app. There is only the possibility to create the standard messages from the app.

play-store-eviewid=gp:3AAOqpTOGSTW8OKzDwDS4DtJLi4WWeosn_JzDU4sH6OTmvBVnnqdTJTa6nvcQrf_Dm1T5qitiCOkWi4S7DJIBFtQ

**Comment by tkeriduN** [comment]
Agreed this is a huge setback for the wordpress mobile apps. A bunch of customers have found this app in the app store and asked med about if they can use it with the sites we've set up for them. Always a huge bummer to have to tell them "No... It's a great app but it doesn't work for anything besides editing pages and standard posts on your site.". And they've hired us for the very reason of creating the custom post/taxonomy structure and we teach them to only use the custom post types.

This has been on the radar here on github for at least 7 years now. IMHO it should be prioritized as blocking, or at least realized that it's **_the main reason_** that people aren't using the mobile apps, even when they really want to.

**Comment by designsimply** [comment]
> When CPT will be supported?
> 
> <sup>3-star review, Jan 13, 2021, 09:02, Samsung Galaxy S10+, App version: 16.4, Android 10</sup>

<sup>play-store-reviewId=gp%3AAOqpTOGL3-aPOUTj_Iylwd_MnDIAmVmHpC1jkCQ-WeEZyBw0lshQ9iMg21area-lbD8wk5nDMsV8x7AG9qaMO6YK&corpus=BETA_FEEDBACK</sup>

**Comment by rezzap** [comment]
From a 3 star app review, Feb. 15th 2021, 07:12:

> Custom Post type not showing on Mobile

**Comment by reginabally** [comment]
From a 3-star app review, Mar 6, 2021, 20:39:

> Nice and easy, but portfolio pages aren't showing as a section which are the central part of my eris theme.

**Comment by thehenrybyrd** [comment]
From a 2-star internal beta review, Jul 8, 2021, 04:37
> It should contains custom posts and metabox when editing/adding them.

**Comment by thabotswana** [comment]
From a 3-star app review, Jul 20, 2021, 08:14:

>All that's missing is the option of Posts Types created in wordpress, as it only allows me to create a standard post.

**Comment by eduardozulian** [comment]
3-star app review on Jul 29, 2021

> Please include custom post types support.

**Comment by kspilarski** [comment]
Missing Portfolio section in WordPress mobile and tablet apps (iOS version 17.9.1 and 18.0). User case in 4242624-zen

**Comment by reginabally** [comment]
From a 1-star iOS app review on Feb 21, 2022:

> There is no possibility to enter or view the recipes, but only the articles and pages

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by Kpudlo** [comment]
It's unfortunate this still isn't possible. CPTs are so common and it makes the app very underpowered, might as well just stick with a browser.
